torchaudio/functional/functional.py

```diff
@@ -77,25 +77,18 @@ def spectrogram(
         onesided (bool, optional): controls whether to return half of results to
             avoid redundancy. Default: ``True``
         return_complex (bool, optional):
```

Comment:
```suggestion
        return_complex (bool or None, optional):
```

Reply:
Here, `None` is used as an indicator to only know if user pass something, so we will keep it as `bool`.